truss/templates/custom/model/model.py

```diff
@@ -12,23 +12,22 @@ def load(self):
         # Load model here and assign to self._model.
         pass
 
-    def preprocess(self, request: Dict) -> Dict:
+    def preprocess(self, model_input: Any) -> Any:
         """
         Incorporate pre-processing required by the model if desired here.
 
         These might be feature transformations that are tightly coupled to the model.
         """
-        return request
+        return model_input
 
-    def postprocess(self, request: Dict) -> Dict:
+    def postprocess(self, model_output: Dict) -> Dict:
```

Comment:
The output type of predict is always a dictionary unless the user changes that behavior (which would mean they are in the model.py file and could change the type). And then that dictionary gets passed through and postprocess returns a dictionary too. Do you mean that it should be

`def postprocess(self, model_output: Dict[str: List]) -> Any:`?

Reply:
technically, we want this to be any type that can be serialized (same as for input). it's not very simple to put that in a typing statement, so any suffices. 

I think the template determines what is idiomatic for users. If we put `Dict` then they'll start using `Dict`. If we put `Any`, they'll do whatever they want. I personally think we need to become less opinionated about things that don't matter and this might be one of them, so `Any` is preferred.

Reply:
In that case, if they are all `Any`, should we just remove type hints altogether from predict?

Reply:
you can, but this will cause lint errors. we need type hinting so that linting doesn't fail. It's also a useful hint to have `Any` in my opinion.